meta.yml

```
@@ -0,0 +1,120 @@
+---
+fullname: Zorn's Lemma
+shortname: zorns-lemma
+organization: coq-community
```

This is missing a `coq-community: true` line that will activate the standard coq-community badges (recent evolution of the README template):

```suggestion
organization: coq-community
coq-community: true
```